### ISSUE TITLE
Re-organize pages and group structures

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-feature-requests#511]
Currently, the pages and groups structure is counter intuitive and not very user friendly at all as it always lead the user away from the content they are viewing or interacting with. This quickly becomes very annoying because you have to keep coming back to the page to see what other contents are on pages or groups.

For example, when you view videos, photos, blogs or forum it always takes users away from the pages. This causes way too many clicks just to move around.

A more user friendly way to organize and make it convenient for the to access any part of pages or groups would be to use a tree like structure. structure on the left hand side for example for each section like video, photos, blogs, forums etc... and the tree would expand or collapse.

**Comment by neil96** [comment]
Thanks @prepperz ,

We will discuss with team and consider adding this feature in next version of pages/groups app.

Regards.

**Comment by dawn2323** [comment]
Exactly! I was wanting to post the same. The User feel and flow is most important. Users tend to get annoyed if they have to click too many times. Especially to back one level should be made simpler!
